components/bl-granim-component/src/index.js

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useRef } from "react";
+
+import { s } from "./lib/granim.min";
+
+const { cn } = BackendlessUI.CSSUtils;
+
+const DIRECTIONS = ["left-right", "diagonal", "top-bottom", "radial", "custom"];
+const BLENDING_MODE = [
```

Comment:
You could create a little helper that map this array to object like

{<mode-name>: 1}

And below when toy looking for mode in this array it could be done through checking property in this map.